.github/workflows/c-cpp.yml

```diff
@@ -12,15 +12,15 @@ jobs:
     strategy:
       matrix:
         os:
-          - ubuntu-18.04
+          - ubuntu-22.04
```

Comment:
consider using ubuntu-latest to avoid similar problems in the future (unless you want to explicitly test on ubuntu 22.04)

Reply:
👍